draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -0,0 +1,1037 @@
+            <t>Using multiple key exchanges CREATE_CHILD_SA is defined.</t>
+            </list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-02</t>
+            <t><list style="symbols">
+        	<t>Use new transform types to negotiate additional key exchanges,
+        	rather than using the KE payloads of IKE SA.</t>
+         	</list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-01</t>
+            <t><list style="symbols">
+     		<t>Use IKE_INTERMEDIATE to perform multiple key exchanges in succession.</t>
+     		<t>Handle fragmentation by keeping the first key exchange (a standard
+     		IKE_SA_INIT with a few extra notifies) small, and encrypting the rest
+     		of the key exchanges.</t>
+    
+     		<t>Simplify the negotiation of the ‘extra’ key exchanges.</t>
+     	    </list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-00</t>
+            <t><list style="symbols">
+    		<t>We added a feature to allow more than one post-quantum key
+            exchange algorithms to be negotiated and used to exchange a post-
+            quantum shared secret.</t>
+    		<t>Instead of relying on TCP encapsulation to deal with IP level
+    		fragmentation, we introduced a new key exchange payload that can
+    		be sent as multiple fragments within IKE_SA_INIT message.</t>
+    	    </list>
+            </t>
+        </section>
+    
+    	<section title="Document Organization" >
+            <t>
+            The remainder of this document is organized as follows.  <xref target="design"/>
+            summarizes design criteria.  <xref target="framework"/> describes how
+    	    multiple key exchanges are performed between two IKE peers and how
+    	    keying materials are derived for both SAs and child SAs.  A summary of alternative
+    	    approaches that have been considered, but later discarded, are described
+    	    in <xref target="altdesign"/>.  <xref target="IANA"/>
+            discusses IANA considerations for the namespaces introduced in this
+    	    document, and lastly <xref target="security"/> discusses security considerations.
+            </t>
+    
+            <t> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", 
+            "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted 
+            as described in BCP 14 <xref target="RFC2119" /> <xref target="RFC8174" /> when, and only when, 
+            they appear in all capitals, as shown here.
+            </t>
+     	</section>
+	</section>
+
+	<section title="Design Criteria" anchor="design">
+        <t>
+        The design of the proposed framework is driven by the
+        following criteria:</t>
+    
+    	<t><list style="hanging" hangIndent="5"><t hangText="1)">
+    	Need for post-quantum cryptography in IPsec.  Quantum computers
+    	might become feasible in the next 5-10 years.  If current
```

Comment:
Don't feel like we should speculate on this time frame, so how about the following?

```suggestion
    	might become feasible in the near future.  If current
```